bundles/org.openhab.binding.luftdateninfo/README.md

```diff
@@ -68,9 +68,9 @@ Perform the following steps to get the appropriate Sensor ID
 luftdaten.things
 
 ```perl
-Thing luftdateninfo:particulate:pm_sensor   "PM Sensor"         [ sensorid=28842]
-Thing luftdateninfo:conditions:cond_sensor  "Condition Sensor"  [ sensorid=28843]
-Thing luftdateninfo:noise:noise_sensor      "Noise Sensor"      [ sensorid=39745]
+Thing luftdateninfo:particulate:pm_sensor   "PM Sensor"         [ sensorid="28842"]
+Thing luftdateninfo:conditions:cond_sensor  "Condition Sensor"  [ sensorid="28843"]
+Thing luftdateninfo:noise:noise_sensor      "Noise Sensor"      [ sensorid="39745"]
```

Comment:
Did you re-add the quotes by intention?

Reply:
Yes, I got an error when removing quotes. I followed this problem and in a previous review I changed the thing-config.xml sensorid from text to int. But I didn't change it in Java code. Now I did this and code and xml fits together. Quotes are removed and it works.